setup-lesson-deps/action.yaml

```diff
@@ -31,15 +31,15 @@ runs:
           if (on_linux) {
             if (Sys.getenv("RSPM") == "") {
               release <- system("lsb_release -c | awk '{print $2}'", intern = TRUE)
-              Sys.setenv("RSPM" = 
-                paste0("https://packagemanager.rstudio.com/all/__linux__/", release, "/latest"))
+              Sys.setenv("RSPM" =
+                paste0("https://packagemanager.posit.co/all/__linux__/", release, "/latest"))
             }
           }
           repos <- list(
             RSPM        = Sys.getenv("RSPM"),
             carpentries = "https://carpentries.r-universe.dev/",
             archive     = "https://carpentries.github.io/drat/",
-            CRAN        = "https://cran.rstudio.com"
+            CRAN        = "https://cran.posit.co"
```

Comment:
an addendum: the CRAN url was _not_ changed, and https://cran.posit.co returns a 404. Lucikly, this repository serves as a _backup_ for when the RSPM is out of date, so no lessons have been affected to my knowledge. I've fixed this in #80. 

Thank you again for opening this pull request!